data/lua/on_event.lua

```diff
@@ -21,20 +21,23 @@ wesnoth.game_events.on_event = function(eventname)
 end
 
 
-local function on_event(eventname, arg1, arg2)
+---Handle an event
+---@param eventname string
+---@param priority? number
```

Comment:
What is the priority used for? Are lower or higher numbers higher priority?

Should either document or deprecate the possibility of having a function argument without a priority argument.

Reply:
I'm not sure. I was planning to deprecate the whole `on_event` module anyway, once that events PR gets finished and merged…

Reply:
Checking the code, it looks like the list is ordered by decreasing priority.